tests/runner.d

```diff
@@ -115,18 +115,18 @@ void test(string filename, string compiler)
     // For some reasons &> is read as & > /dev/null causing the compiler to return 0.
     version (Posix) if(!expectedToCompile || true) command ~= " 2> /dev/null 1> /dev/null";
     
-    auto retval = system(command);
-    if (expectedToCompile && retval != 0) {
+    auto retval = executeShell(command);
```

Comment:
retval stand for return value. executeShell seems to return some kind of struct that is richer that just the return value of the program. Can you rename this to something more appropriate ?